azext_iot/central/providers/device_provider.py

```diff
@@ -246,3 +262,28 @@ def get_all_registration_info(
         ]
 
         return result
+
+    def get_registration_summary(self, central_dns_suffix="azureiotcentral.com"):
+        registration_summary = {}
+
+        devices = self.list_devices(central_dns_suffix=central_dns_suffix)
+        for device in devices.values():
+            if not device.simulated:
+                registration_summary[device.device_status.name] = (
+                    registration_summary.get(device.device_status.name, 0) + 1
+                )
+        return registration_summary
+
+    def print_limited_devices(self, devices_to_display, device_collection):
```

Comment:
this also looks like a private method

Reply:
Actually I don't think this method should be in the provider, it should be in the `commands_*` layer since its not really doing work, its just breaking up the results